### PR TITLE
This is to push a new Hello World Page for fun

### PR DESCRIPTION
**Pull request by AnitaTT**
:))))))

### CODE REVIEW

File: slack-announcement-scheduler/test/controllers/welcome_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class WelcomeControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (ChiuWeiYi) [inline slack-announcement-scheduler/test/controllers/welcome_controller_test.rb:6]: why comments?

File: slack-announcement-scheduler/app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
+    def say
+    end
```
Comment (ChiuWeiYi) [inline slack-announcement-scheduler/app/controllers/welcome_controller.rb]:
```suggestion
    def say_hello
    end
```

**Review by ChiuWeiYi — COMMENTED** [review]
generally good, just one question

File: slack-announcement-scheduler/app/controllers/welcome_controller.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
+    def say
```
Comment (ChihHaoTW) [inline slack-announcement-scheduler/app/controllers/welcome_controller.rb]:
```suggestion
    def say_hello
```

File: slack-announcement-scheduler/config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,3 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "welcome/say_hello" => "welcome#say"
```
Comment (ChihHaoTW) [inline slack-announcement-scheduler/config/routes.rb]:
```suggestion
  get "welcome/say_hello" => "welcome#say_hello"
```

**Review by ChihHaoTW — APPROVED** [review]
LGTM

**Pull request merged**